grub-mkconfig.md

```diff
@@ -0,0 +1,8 @@
+# grub-mkconfig
```

Comment:
Based on the command help page, we can add a first example with just:
```
grub-mkconfig
```
And the generated file is send to the `stdout`, looks like this could be used as a dry run (without side effects in the OS).

Also we can add the help just to let the user know he can more info in the CLI:
```
grub-mkconfig --help
```

Reply:
@Managor even though `--help` is not a particularly useful thing to document, especially because we don't document it in many other pages, if there's nothing else to document and we're well below the 8-example limit, we usually add options like `--help` (if the command has it, which this one does :wink:).